docs/modules/ROOT/pages/migration-guide/3.0.0.adoc

```diff
@@ -1,4 +1,4 @@
-= Camel Quarkus 3.0.0 Migration Guide
+x`= Camel Quarkus 3.0.0 Migration Guide
```

Comment:
This is typo?

Reply:
@zhfeng Nice catch 
there's also a grpc test pom.xml that I need to rollback
let me do this.